README.md

```diff
@@ -478,6 +478,7 @@ Services to securely store your Docker images.
 - [libcompose](https://github.com/docker/libcompose) - Go library for Docker Compose.
 - [sbt-docker-compose](https://github.com/Tapad/sbt-docker-compose) - Integrates Docker Compose functionality into sbt by [@kurtkopchik](https://github.com/kurtkopchik/)
 - [sbt-docker](https://github.com/marcuslonnberg/sbt-docker) - Create Docker images directly from sbt by [@marcuslonnberg](https://github.com/marcuslonnberg)
+- [docker-compose-ui](https://github.com/francescou/docker-compose-ui) - Manage docker-compose via HTTP. docker-compose-ui runs in a Docker container, mounts the hosts docker socket and exposes a RESTful API and AngularJS GUI
```

Comment:
Thank you for contributing. Could you add it to the section "WEB" instead, please ?

Also what do you think of this:
```suggestion
- [docker-compose-ui](https://github.com/francescou/docker-compose-ui) - Docker Compose UI is a web interface for Docker Compose.
```
The reason for this suggestion is the project itself describe it that way. [See here](http://francescou.github.io/docker-compose-ui/)

Reply:
Hi @binarymist 

Thank you for elaborating on description. It will be useful for other people as well.

You can even make a pull request to the original project :-)

Reply:
I thought it was the original project?